python/tvm/relay/op/transform.py

```diff
@@ -180,3 +180,28 @@ def full_like(data, fill_value):
         The resulting tensor.
     """
     return _make.full_like(data, fill_value)
+
+
+def split(data, indices_or_sections, axis=None, equal_split=False):
+    """Split input tensor along axis by sections or indices.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The source array.
+
+    indices_or_sections : tuple of int
+        The indices or number of sections to split.
+
+    axis : int, optional
+        The axis over which to split.
+
+    equal_split : boolean
+        To split into indices_or_section[0] sections.
```

Comment:
could you explain a bit more for `indices_or_sections` and `equal_split`? it seems to be not so clear to users.